ngraph/frontend/onnx_import/src/op/non_zero.cpp

```diff
@@ -17,7 +17,11 @@ namespace ngraph
             {
                 OutputVector non_zero(const Node& node)
                 {
-                    const auto data = node.get_ng_inputs().at(0);
+                    auto data = node.get_ng_inputs().at(0);
+                    if (data.get_element_type() == element::boolean)
+                    {
+                        data = std::make_shared<default_opset::Convert>(data, element::u8);
```

Comment:
Is there an onnx importer test for boolean inputs that validates it?

Reply:
I have no information about such case but bool type is supported in the ONNX spec while OV NonZero accept only numeric types.